tests/tunneldigger.py

```diff
@@ -33,7 +33,7 @@ def setup_template():
             raise RuntimeError("failed to start container")
 
     container.attach_wait(lxc.attach_run_command, ["dhclient", "eth0"])
-    check_ping(container, '8.8.8.8', 10)
+    check_ping(container, 'ccc.de', 10)
```

Comment:
Could we ping instead `google-public-dns-a.google.com`? I believe it might have higher uptime than `ccc.de`. ;-)